.github/workflows/import-strings.yml

```diff
@@ -0,0 +1,39 @@
+name: Create a PR with changes after importing strings
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        persist-credentials: false 
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Get the current date for PR title
+      run: echo "current_date=$(date +"%m-%d-%Y")" >> $GITHUB_ENV
+    - name: Run script to import strings
+      run: sh ./test-fixtures/import-strings.sh
+    - name: Update new strings
+      run: |-
+        git diff || (git add Blockzilla/*/*.lproj/* Blockzilla/*.lproj/*)
```

Comment:
These are the kind of files I have seen being modified.. am I missing some more?

Reply:
I think that should catch them all